pkg/builder/builder.go

```diff
@@ -368,10 +368,13 @@ func (b generalBuilder) build(ctx context.Context, pw progresswriter.Writer) err
 }
 
 func (b generalBuilder) checkIfNeedBuild(ctx context.Context) bool {
+	if ir.DefaultGraph.HTTP != nil || ir.DefaultGraph.Repo != (types.RepoInfo{}) {
```

Comment:
`Repo` will only add an image label. `envd-server` will use a init container to pull and mount to the container.

So I guess this should not trigger a rebuild.

Reply:
Same opinion here.